apps/vr-tests-react-components/package.json

```diff
@@ -43,6 +43,7 @@
     "@fluentui/react-slider": "^9.0.8",
     "@fluentui/react-spinner": "^9.0.8",
     "@fluentui/react-spinbutton": "^9.0.6",
+    "@fluentui/react-storybook-addon": "^9.0.0-rc.1",
```

Comment:
pre-releases should never use carets as those resolution is non deterministic ( alphabetical order based )
```suggestion
    "@fluentui/react-storybook-addon": "9.0.0-rc.1",
```

Reply:
Setting it to a pinned version currently fails syncpack:
```
✕ @fluentui/react-storybook-addon
- ^9.0.0-rc.1 in dependencies of @fluentui/public-docsite-v9
- 9.0.0-rc.1 in dependencies of @fluentui/vr-tests-react-components
- ^9.0.0-rc.1 in devDependencies of @fluentui/react-components

Reply:
okay, lets address this in separate PR

Reply:
Made a separate issue: https://github.com/microsoft/fluentui/issues/25490